packages/governance/src/contractGovernance/paramManager.js

```diff
@@ -163,6 +164,21 @@ const makeParamManagerBuilder = zoe => {
     return builder;
   };
 
+  /** @type {(name: string, value: string, builder: ParamManagerBuilder) => ParamManagerBuilder} */
+  const addJson = (name, value, builder) => {
+    const assertJson = v => {
+      try {
+        assert.typeof(v, 'string');
+        JSON.parse(v);
+      } catch (e) {
+        throw Error(`Expected a json string. "${v}" is not valid json.`);
+      }
+      return true;
```

Comment:
this is consistent with the others, but why do any return `true`? IMO `assert` should always return void and sometimes throw.

Reply:
done